api/proxy.config.ts

```diff
@@ -1,113 +1,115 @@
+  applyProxy(app, {
+    middlewares: [bodyParser.json()],
+    onReq: modifyRequest,
+    onRes: handleElasticSearchResponse,
+    rewrite: false,
+    source: '/data/internal/searchCases',
+    target: getConfigValue(proxiedReferences.SERVICES_CCD_COMPONENT_API_PATH),
+  });
 
-    applyProxy(app, {
-        rewrite: true,
-        rewriteUrl: '/addresses',
-        source: '/api/addresses',
-        target: getConfigValue(SERVICES_CCD_COMPONENT_API_PATH),
-    })
+  applyProxy(app, {
+    rewrite: true,
+    rewriteUrl: '/addresses',
+    source: '/api/addresses',
+    target: getConfigValue(proxiedReferences.SERVICES_CCD_COMPONENT_API_PATH),
+  });
 
-    applyProxy(app, {
-        onReq: amendedJurisdictions.checkCachedJurisdictions,
-        onRes: amendedJurisdictions.getJurisdictions,
-        rewrite: false,
-        source: '/aggregated',
-        target: getConfigValue(SERVICES_CCD_COMPONENT_API_PATH),
-    })
+  applyProxy(app, {
+    onReq: checkCachedJurisdictions,
+    onRes: getJurisdictions,
+    rewrite: false,
+    source: '/aggregated',
+    target: getConfigValue(proxiedReferences.SERVICES_CCD_COMPONENT_API_PATH),
+  });
 
-    applyProxy(app, {
-        rewrite: false,
-        source: '/icp',
-        target: getConfigValue(SERVICES_ICP_API_URL),
-        ws: true,
-    })
+  applyProxy(app, {
+    rewrite: false,
+    source: '/icp',
+    target: getConfigValue(proxiedReferences.SERVICES_ICP_API_URL),
+    ws: true,
+  });
 
-    applyProxy(app, {
-        rewrite: true,
-        rewriteUrl: '/api',
-        source: '/em-anno',
-        target: getConfigValue(SERVICES_EM_ANNO_API_URL),
-    })
+  applyProxy(app, {
+    rewrite: true,
+    rewriteUrl: '/api',
+    source: '/em-anno',
+    target: getConfigValue(proxiedReferences.SERVICES_EM_ANNO_API_URL),
+  });
 
-    applyProxy(app, {
-        rewrite: false,
-        source: [
-            '/api/markups',
-            '/api/redaction',
-        ],
-        target: getConfigValue(SERVICES_MARKUP_API_URL),
-    })
+  applyProxy(app, {
+    rewrite: false,
+    source: [
+      '/api/markups',
+      '/api/redaction',
+    ],
+    target: getConfigValue(proxiedReferences.SERVICES_MARKUP_API_URL),
+  });
 
-    applyProxy(app, {
-        rewrite: true,
-        rewriteUrl: '',
-        source: '/payments',
-        target: getConfigValue(SERVICES_PAYMENTS_URL),
-    })
+  applyProxy(app, {
+    rewrite: true,
+    rewriteUrl: '',
+    source: '/payments',
+    target: getConfigValue(proxiedReferences.SERVICES_PAYMENTS_URL),
+  });
 
-    /**
-     * Commenting this out as it's completely bypassing the API this way,
-     * which is not we want right now. If that's to be the long-term
-     * solution, we'll need to move the logic around adding actions out
-     * of the node layer.
-     */
-    // applyProxy(app, {
-    //     rewrite: true,
-    //     rewriteUrl: '',
-    //     source: '/workallocation',
-    //     target: 'http://localhost:8080',
-    // })
-}
+  /**
```

Comment:
(Nitpick)
Do you think this will ever be used? If not or not in the near future I'd be tempted to remove it completely.

Reply:
I simply changed the indenting on this but it's a good question. @udaydenduluri33, any thoughts on this? Can I delete it rathe than it being commented out?